trendmicro_apex/icon_trendmicro_apex/actions/add_to_udso_list/action.py

```diff
@@ -0,0 +1,93 @@
+        payload_type = params.get(Input.DATA_TYPE)
+        num_days = params.get(Input.EXPIRY_DATE, 30)
+        payload_expiry_date = get_expiration_utc_date_string(int(num_days))
+
+        if payload_type == 'FILE_SHA1':
+            if len(payload_content) > 40:
+                self.logger.warning("SHA passed in too long, truncated to 40 characters")
+            payload_content = payload_content[:40]
+        elif payload_type == 'IP':
+            try:
+                test_ip = ipaddress.ip_address(payload_content)
+            except ValueError:
+                raise PluginException(cause='Invalid IP provided when sending data type IP',
+                                      assistance='Please verify the fields DATA_TYPE and CONTENT')
+        elif payload_type == 'URL':
+            # url regex's provided by urlregex.com
+            start_of_content = payload_content[:6].lower()
+            if not start_of_content == "https:" and not start_of_content[:5] == "http:":
+                raise PluginException(cause='HTTP or HTTPS needed when submitting data type URL',
+                                      assistance='Please start the URL with http or https')
+            elif not re.search('^http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+$',
+                payload_content):
+                raise PluginException(cause='Improperly formatted URL provided when sending data type URL',
+                                      assistance='Please verify the fields DATA_TYPE and CONTENT')
+            if len(payload_content) > 2046:
+                self.logger.warning("URL passed in too long, truncated to 2046 characters")
+            payload_content = payload_content[:2046]
+        elif payload_type == 'DOMAIN':
+            if not re.search('^(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+$',
+                             payload_content):
+                raise PluginException(cause='Improperly formatted DOMAIN provided when sending data type DOMAIN',
+                                      assistance='Please verify the fields DATA_TYPE and CONTENT')
+
+        payload = {
+            "param": {
+                "content": payload_content,
+                "expiration_utc_date": payload_expiry_date,
+                "notes": payload_notes,
+                "scan_action": payload_scan_action.lower(),
+                "type": payload_type.lower()
+            }
+        }
+        json_payload = json.dumps(payload)
+        jwt_token = self.connection.create_jwt_token(self.api_path, self.api_http_method, json_payload)
+        request_url = self.connection.hostname + self.api_path
+
+        response = None
+        try:
+            response = requests.put(request_url, headers=self.connection.header_dict, data=json_payload, verify=False)
+            response.raise_for_status()
+            return {Output.SUCCESS : bool(not(response == None))}
```

Comment:
```suggestion
            return {Output.SUCCESS : response is not None}
```
Easier to read and the proper way to check if something is null is to use `is`